app/code/Magento/Captcha/view/frontend/templates/default.phtml

```diff
@@ -5,29 +5,29 @@
  */
 
 /** @var \Magento\Captcha\Block\Captcha\DefaultCaptcha $block */
-
+/** @var \Magento\Framework\Escaper $escaper */
 /** @var \Magento\Captcha\Model\DefaultModel $captcha */
```

Comment:
Could you combine it to a single phpdoc block?

Reply:
fixed

Reply:
Trigger removing "Progress: needs update" lable